src/alloy/chart/templates/alloy-config.yaml

```diff
@@ -0,0 +1,213 @@
+      role = "pod"
+    }
+    discovery.relabel "pod_logs" {
+      targets = discovery.kubernetes.pod_logs.targets
+      rule {
+        source_labels = ["__meta_kubernetes_pod_controller_name"]
+        regex         = "([0-9a-z-.]+?)(-[0-9a-f]{8,10})?"
+        target_label  = "__tmp_controller_name"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_label_app_kubernetes_io_name", "__meta_kubernetes_pod_label_app", "__tmp_controller_name", "__meta_kubernetes_pod_name"]
+        regex         = "^;*([^;]+)(;.*)?$"
+        target_label  = "app"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_label_app_kubernetes_io_instance", "__meta_kubernetes_pod_label_instance"]
+        regex         = "^;*([^;]+)(;.*)?$"
+        target_label  = "instance"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_label_app_kubernetes_io_component", "__meta_kubernetes_pod_label_component"]
+        regex         = "^;*([^;]+)(;.*)?$"
+        target_label  = "component"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_node_name"]
+        target_label  = "node_name"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_namespace"]
+        target_label  = "namespace"
+      }
+      rule {
+        source_labels = ["namespace", "app"]
+        separator     = "/"
+        target_label  = "job"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_name"]
+        target_label  = "pod"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_container_name"]
+        target_label  = "container"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_uid", "__meta_kubernetes_pod_container_name"]
+        separator     = "/"
+        target_label  = "__path__"
+        replacement   = "/var/log/pods/*$1/*.log"
+      }
+      rule {
+        source_labels = ["__meta_kubernetes_pod_annotationpresent_kubernetes_io_config_hash", "__meta_kubernetes_pod_annotation_kubernetes_io_config_hash", "__meta_kubernetes_pod_container_name"]
+        separator     = "/"
+        regex         = "true/(.*)"
+        target_label  = "__path__"
+        replacement   = "/var/log/pods/*$1/*.log"
+      }
+      rule {
+        source_labels = ["job"]
+        regex         = "alloy/alloy" // Drop any logs from Alloy itself to prevent log loops
+        action        = "drop"
+      }
+    }
+    local.file_match "pod_logs" {
+      path_targets = discovery.relabel.pod_logs.output
+    }
+
+    // K8s Logs (ex: audit log)
+    local.file_match "k8s_logs" {
+      path_targets = [{
+        __path__  = "/var/log/kubernetes/**/*.log",
+        job       = "kubernetes-logs",
+        host = env("HOSTNAME"),
+      }]
+    }
+
+    // Forward all logs to Destinations
+    loki.source.file "node_logs" {
+      targets    = local.file_match.node_logs.targets
+      legacy_positions_file = "/run/promtail/positions.yaml"
```

Comment:
Pushed up to make sure this was tracked but it does not appear to be working currently - seeing duplicate logs for node/pods and no indication it used the promtail position file.

Reply:
In further testing logs did appear to properly be using the positions file across the upgrade. I did not note any changes in the latest patch version of Alloy so unsure if I just had faulty testing previously or if something else was at play.